libs/pulumi/src/executors/up/executor.ts

```diff
@@ -26,39 +26,36 @@ export default async function runUpExecutor(
             options.buildTarget ?? 'build'
         }:production`,
     )
-    // Build project to be deployed
-    for await (const s of await runExecutor(
-        {
-            project: options.targetProjectName,
-            target: options.buildTarget ?? 'build',
-            configuration: 'production',
-        },
-        {},
-        context,
-    )) {
-        if (!s.success) {
-            return {
-                success: false,
-            }
-        }
-    }
 
-    for (const additionalBuildTarget of options.additionalBuildTargets || []) {
+    const buildTargets = options.buildTargets ?? []
+    const deprecatedBuildTarget = options.buildTarget ? [{
+        project: options.targetProjectName,
+        target: options.buildTarget ?? 'build',
+        configuration: 'production',
+    }] : []
+    const deprecatedAdditionalBuildTargets
+        = options.additionalBuildTargets ?? []
+
+    for (const buildTarget of [
+        ...buildTargets,
+        ...deprecatedBuildTarget,
```

Comment:
TBH i'd just make it a breaking change, the lib isn't at v1 so we can make breaking changes

Reply:
I know, I just don't really want to redo what's already in workspace.json in other projects 😛

Reply:
An nx migration would help